sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -296,17 +296,18 @@ await using (var scope = await EventHubScope.CreateAsync(1))
                 var clientOptions = new EventHubClientOptions
                 {
                     Proxy = new WebProxy("http://1.2.3.4:9999"),
-                    TransportType = TransportType.AmqpWebSockets
+                    TransportType = TransportType.AmqpWebSockets,
+                    RetryOptions = new RetryOptions { TryTimeout = TimeSpan.FromMinutes(2) }
                 };
 
                 await using (var client = new EventHubClient(connectionString))
                 await using (var invalidProxyClient = new EventHubClient(connectionString, clientOptions))
                 {
                     var partition = (await client.GetPartitionIdsAsync()).First();
 
-                    Assert.That(async () => await invalidProxyClient.GetPartitionIdsAsync(), Throws.InstanceOf<WebSocketException>());
-                    Assert.That(async () => await invalidProxyClient.GetPropertiesAsync(), Throws.InstanceOf<WebSocketException>());
-                    Assert.That(async () => await invalidProxyClient.GetPartitionPropertiesAsync(partition), Throws.InstanceOf<WebSocketException>());
+                    Assert.That(async () => await invalidProxyClient.GetPartitionIdsAsync(), Throws.InstanceOf<WebSocketException>().Or.InstanceOf<TimeoutException>());
```

Comment:
Linux and macOS are consistently throwing a Timeout, even when this test is run alone.   This is likely why there was a conditional around the proxy tests in the track one code.   I think loosening the criteria is valid here on all platforms.